HGustavs/LenaSYS
# Merge #12600 and #12603

Bigger merge which includes the following solutions.

Add IE-entities. #12600
Add IE-Inheritance #12603
Add an IE option in the variant editor #12813

**What has been done**
- IE entity added in both toolbar plus option panel.
- IE inheritance was added in both toolbar plus option panel.
- Subtoolbar functionality for IE elements fixed. 
- IE added in Edit dugga so the functionality can be hidden/displayed. 

**How to test**
Test so that functionality works in the same way as UML in diagram.
It should also be possible to select/disable IE from Edit dugga.
